homeassistant/components/vallox/sensor.py

```diff
@@ -123,15 +121,27 @@ class ValloxSensorEntityDescription(SensorEntityDescription):
     """Describes Vallox sensor entity."""
 
     metric_key: str | None = None
-    sensor_type: type[ValloxSensor] = ValloxSensor
+    entity_type: type[ValloxSensorEntity] = ValloxSensorEntity
+    round_ndigits: int | None = None
 
 
-SENSORS: tuple[ValloxSensorEntityDescription, ...] = (
+class ValloxProfileSensor(ValloxSensorEntity):
```

Comment:
Just a nitpick:
The structure of the file from top to bottom is now:
1. Sensor Entity child class definitions.
2. ValloxSensorEntityDescription dataclass definition.
3. One more Sensor Entity child class definition (ValloxProfileSensor).
4. SENSOR_ENTITIES.

If we add ValloxProfileSensor back to where it originally was to retain consistent structure.

Reply:
fixed